src/main/webapp/WEB-INF/templates/search.jsp

```diff
@@ -12,7 +13,23 @@
     <t:base/>
     <div class="gcse-searchbox-only" data-resultsUrl="/search"></div>
     <div class="srp-container">
-      <div class="gcse-searchresults-only"></div>
+      <div class="srp-child">
+        <div class="gcse-searchresults-only"></div>
+      </div>
+      <div class="srp-child">
+        <c:forEach items='${requestScope["cardList"]}' var="knowledgeCard">
+          <div class="card">
+            <div class="card-content">
+              <a href="${knowledgeCard.getLink()}" title="card-source">
+                <span class="card-title">${knowledgeCard.getTitle()}</span>
+              </a>
+              <p>${knowledgeCard.getCode()}</p>
+              <span class="card-title">Description</span>
```

Comment:
style nit: When you hover over "Description", a pointer appears instead of a cursor since this shouldn't be clickable. 

I think this is because you're using "card-title" class here which is the same class as you use for the actual hyperlinked knowledge card title. I think this description span should have a different unique class.